client/client.go

```diff
@@ -27,8 +29,10 @@ const (
 // Client is the implementation of HTTP1 client which sets body as a
 // stream.
 type Client struct {
-	dialer    Dialer
-	tlsConfig *tls.Config
+	dialer           Dialer
+	tlsConfig        *tls.Config
```

Comment:
This is not required anymore then

Reply:
But we still pass there `defaultTLSConfig = &tls.Config{InsecureSkipVerify: true}`
And it's cloned for new configs.
Should I remove defaultTLSConfig then too?

Reply:
I would really appreciate that :) Yes, the whole purpose of this defaultTLSConfig is to propagate something initialized for `dialer` field. Since we have a map there, it is absolutely redundand

Reply:
done